lms/djangoapps/ccx/views.py

```diff
@@ -407,6 +424,47 @@ def ccx_schedule(request, course, ccx=None):  # pylint: disable=unused-argument
     return HttpResponse(json_schedule, mimetype='application/json')
 
 
+def _get_valid_student_email(identifier):
+    """
+    helper function to get an user email
+    from an identifier and validate it
```

Comment:
I did not add a proper description in this function because I declared it "private", but I will take care of that